subprojects/code-quality/src/testFixtures/groovy/org/gradle/quality/integtest/fixtures/CheckstyleCoverage.groovy

```diff
@@ -21,9 +21,10 @@ import org.gradle.util.VersionNumber
 
 class CheckstyleCoverage {
     // Note, this only work for major.minor versions
-    final static List<String> ALL = ['6.12', '6.9', '6.5', '6.0', '5.5', CheckstylePlugin.DEFAULT_CHECKSTYLE_VERSION].asImmutable()
+    final static List<String> ALL = ['7.4', '7.0', '6.12', '6.9', '6.2', CheckstylePlugin.DEFAULT_CHECKSTYLE_VERSION].asImmutable()
```

Comment:
Is there a specific reason why we changed the order of versions from latest to oldest?